woocommerce/woocommerce-gateway-stripe
# UPE: Extract Stripe Api wrapper to a separate file

# Changes proposed in this Pull Request:

Issue: 
Fixes #

Extracts the Stripe API wrapper to a separate file


# Testing instructions

- Run npm run start:webpack
- Go to http://localhost:8082/checkout/, select Stripe UPE, and check that the Credit Card form loads correctly.
